subprojects/wrapper/wrapper.gradle

```diff
@@ -38,6 +38,8 @@ tasks.register("executableJar", Jar) {
     reproducibleFileOrder = true
     preserveFileTimestamps = false
     archiveName = 'gradle-wrapper.jar'
+    fileMode = 0644
+    dirMode = 755
     from sourceSets.main.output
```

Comment:
❓ Why not use `0600` and `0700` by default then?

Reply:
I don't care too much. What is the advantage of `0600` and `0700`?

This doesn't really have any effect as long as you don't unzip the jar. I you unzip, I would leave the entries readable for everyone.